nodejs/week2/search-engine-warmup.js

```diff
@@ -0,0 +1,78 @@
+import express from 'express'
+import fs from 'fs';
+
+const app = express();
+const router = express.Router();
+const port = process.env.PORT || 9999;
+
+app.use(express.json());
+
+const documents = JSON.parse(fs.readFileSync('./documents.json', 'utf-8'));
+
+app.get("/", (req, res) => {
+    res.send("This is a search engine");
+});
+
+router.get("/search", (req, res) => {
```

Comment:
Optionally you could convert the search query and the strings inside the documents to lowercase so that the search is case insensitive.